src/components/CatergoryList/CategoryList.jsx

```diff
@@ -0,0 +1,34 @@
+import { useCategory } from "contexts";
+import "./category-list.css";
+
+const CategoryList = () => {
+  const { categoryState } = useCategory();
+  console.log(categoryState);
+  return (
+    <div className="categories-container mt-7">
+      {categoryState.categories.map((category) => {
```

Comment:
You could add a loader here till the category state has items to get better user experience.